lib/validator.js

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const validation = require('bedrock-validation');
+
+exports.validator = ({bodySchema, querySchema}) => {
+  return (req, res, next) => {
```

Comment:
please add validation above L9 (not in the middleware) to ensure that at least one of `bodySchema` or `querySchema` is defined.  I note that if someone were to pass in params incorrectly then the validator becomes a noop and we fall through to L22.
```
if(!(bodySchema || querySchema)) {
  throw new TypeError('One of the following parameters is required: "bodySchema", "querySchema".');
}
```
```